src/runtime_src/core/include/xrt.h

```diff
@@ -129,6 +129,7 @@ struct xclDeviceInfo2 {
   unsigned short mVccIntVol;
   unsigned short mVccIntCurr;
   unsigned short mNumCDMA;
+  unsigned long long m3v3PexCurr;
```

Comment:
i thought i have mentioned that no change should be made in this data structure in xrt.h since everything moves to sysfs node? Please let me know why you do need to add more things to it.